test/run_test.py

```diff
@@ -1182,14 +1180,16 @@ def parse_args():
             or TEST_WITH_ASAN
             or (
                 strtobool(os.environ.get("TD_DISTRIBUTED", "False"))
-                and os.getenv("TEST_CONFIG") == "distributed"
+                and TEST_CONFIG == "distributed"
                 and TEST_CUDA
             )
             or (IS_WINDOWS and not TEST_CUDA)
+            or TEST_CONFIG == "nogpu_AVX512"
```

Comment:
Nit: If you plan to continue rolling this out to more test config, it might be more readable to move them into a LIST and have `TEST_CONFIG in LIST_OF_TEST_CONFIGS_WITH_TD_SUPPORT` instead I think

Reply:
I'll do it in the future if it I end up adding more or statements like this